financialaid/migrations/0016_load_country_income_thresholds.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-10-12 15:12
+from __future__ import unicode_literals
+
+import json
+import os
+
+from django.conf import settings
+from django.db import migrations
+
+from financialaid.constants import DEFAULT_INCOME_THRESHOLD
+
+
+def load_country_income_thresholds(apps, schema_editor):
+    """
+    Loads data from financialaid/fixtures/country_income_threshold_data.json into the database.
+
+    Note: Instead of simply using call_command("loaddata", "country_income_threshold_data") to load the fixture,
+    this manually creates database objects from data in the fixture. This way we can set the correct created_on
+    and updated_on timestamps as well as set the income_threshold to DEFAULT_INCOME_THRESHOLD should this value
+    change from the default income_treshold in the fixture.
+    """
+    CountryIncomeThreshold = apps.get_model("financialaid", "CountryIncomeThreshold")
+    fixture_path = os.path.join(settings.BASE_DIR, "financialaid/fixtures/country_income_threshold_data.json")
+    with open(fixture_path, "r") as f:
+        country_data = json.loads(f.read())
+    for country in country_data:
+        CountryIncomeThreshold.objects.create(
+            country_code=country["fields"]["country_code"],
+            income_threshold=DEFAULT_INCOME_THRESHOLD
+        )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('financialaid', '0015_countryincomethreshold'),
+    ]
+
+    operations = [
+        migrations.RunPython(load_country_income_thresholds),
+    ]
```

Comment:
can you try to revert this migration? if it does work we do not need to add a "south" migration because this is the first data we load, otherwise we need one.